feathr-example/scripts/poolupdate.sh

```diff
@@ -0,0 +1,32 @@
+subscription_id="<SUBCRIPTION_ID>"
```

Comment:
Maybe add some description about what this is supposed to do?

Reply:
Not sure about this, I thought the variable and value (that is to be replaced) are descriptive enough for its usage.